### DISCUSSION TITLE
I would like to translate the documentation into Portuguese.

### DISCUSSION THREAD

**Discussion by hknsh** [biomejs/biome#1063] [General]
Hello!
Biome is one of my favorite projects for web development, and I'm using it in various side projects. As a contribution, I would love to translate the documentation into Portuguese to attract more Brazilians to the project!

**Comment by Conaclos** [comment]
You are welcome! Feel free to open an umbrella issue to track which pages have to be translated. Some other devs could help you to translate and review the works from every one. [See this comment](https://github.com/biomejs/biome/issues/880#issuecomment-1826477936) for an example.

Note that we only translate pages that are not automatically generated.
